doc/Overview/Mvux/States.md

```diff
@@ -130,7 +130,7 @@ In this scenario, the `DataContext` is set to an instance of the `BindableSlider
 
 ### Change data of a state
 
-To manually update the current value of a state, use its `Update` method.  
+To manually update the current value of a state, use its `Update` operator.  
```

Comment:
Why "operator", isn't Update a method, in which case use method

Reply:
@nickrandolph  @weitzhandler  the "operators" is for a "feed that is wrapping another feed in order to apply a given operation". They are typically declared fluently like `MyFeedA.***Where(...)***` or `MyFeedA.***Select(...)***`. There are to oppose to the "sources" feeds that are producing values based on the internal own logic, for instance `Feed.Async`.

And in that case `method` is more appropriate as it's only a plain method that modifies the internal state of the `IState`. It does not return a feed.

Reply:
Thanks for the input David!